lib/launcher.js

```diff
@@ -153,6 +154,26 @@ var init = function(configFile, additionalConfig) {
       }
     });
   }).then(function() {
+    // 3) If we're in `elementExplorer` mode, run only that.
+    if (config.elementExplorer) {
+      if (config.multiCapabilities.length != 1) {
+        throw new Error('Must specify only 1 browser while using elementExplorer');
```

Comment:
Ahh, I see what you mean. I did mark the explorer framework as private so people shouldn't be messing around with it...but I'll change the conditional to safeguard it too.